sim/hunter/serpent_sting.go

```diff
@@ -122,3 +121,71 @@ func (hunter *Hunter) registerSerpentStingSpell() {
 		}
 	}
 }
+
+/*
+ 	STRINGS OF FATE's SERPENT STING
+*/
+
+// https://www.wowhead.com/classic-ptr/spell=1232982/serpent-sting
+func (hunter *Hunter) getSoFSerpentStingConfig(stacks int) core.SpellConfig {
+	spellId := [5]int32{0, 1232979, 1232980, 1232981, 1232982}[stacks]
+	numTicks := [5]int32{0, 7, 14, 21, 28}[stacks]
+	baseDamage := [5]float64{0, 777, 1554, 2331, 3108}[stacks] / float64(numTicks)
+	
+
+	return core.SpellConfig{
+		ClassSpellMask: ClassSpellMask_HunterSoFSerpentSting,
+		ActionID:       core.ActionID{SpellID: spellId},
+		SpellSchool:    core.SpellSchoolNature,
+		CastType:       proto.CastType_CastTypeUnknown,
+		ProcMask:       core.ProcMaskEmpty,
+		Flags:          core.SpellFlagPureDot | core.SpellFlagPoison | SpellFlagSting,
```

Comment:
This should probably have `core.SpeSpellFlagPassiveSpell` and `SpellFlagNoOnCastComplete` since it's triggered by a proc